rdkit/Chem/SaltRemover.py

```diff
@@ -146,6 +199,31 @@ def StripMol(self, mol, dontRemoveEverything=False):
     2
 
     """
+    strippedMol = self._StripMol(mol, dontRemoveEverything)
+    return strippedMol.mol
+
+  def StripMolWithDeleted(self, mol, dontRemoveEverything=False):
+    """
+    Strips given molecule and returns it, with the fragments which have been deleted.
+
+    >>> remover = SaltRemover(defnData="[Cl,Br]")
+    >>> len(remover.salts)
+    1
+
+    >>> mol = Chem.MolFromSmiles('CN(C)C.Cl')
+    >>> res, deleted = remover.StripMolWithDeleted(mol)
+    >>> res.GetNumAtoms()
+    4
+    >>> len(deleted)
+    1
+    >>> deleted[0].GetNumAtoms()
+    1
+    >>> Chem.MolToSmiles(deleted[0])
+    'Cl'
+    """
```

Comment:
As you added an unit test file, I would use the doctest to demonstrate the use of the function; more documentation than test. Something along the lines of:
```
    >>> remover = SaltRemover(defnData="[Cl,Br]")
    >>> mol = Chem.MolFromSmiles('CN(C)C.Cl.Br')
    >>> res, deleted = remover.StripMolWithDeleted(mol)
    >>> Chem.MolToSmiles(res)
    'CN(C)C'
    >>> [Chem.MolToSmiles(m) for m in deleted]
    ['Cl', 'Br']
```
However, don't change the old doctests.